contracts/modules/validators/K1Validator.sol

```diff
@@ -214,6 +198,24 @@ contract K1Validator is IValidator, ERC7739Validator {
                                      INTERNAL
     //////////////////////////////////////////////////////////////////////////*/
 
+    /// @notice Recovers the signer from a signature
+    /// @param hash The hash of the data to validate
+    /// @param signature The signature data
+    /// @return The recovered signer address
+    /// @notice tryRecover returns address(0) on invalid signature
+    function _recoverSigner(bytes32 hash, bytes calldata signature) internal view returns (address) {
+        return hash.tryRecover(signature);
+    }
+
+    /// @notice Recovers the signer from an Ethereum signed message
+    /// @param hash The hash of the data to validate
+    /// @param signature The signature data
+    /// @return The recovered signer address
+    /// @notice tryRecover returns address(0) on invalid signature
+    function _recoverSignerFromEthSignedMessage(bytes32 hash, bytes calldata signature) internal view returns (address) {
```

Comment:
we do not need this function

use `_recoverSigner(hash.toEthSignedMessageHash(), signature)` instead

Reply:
ok

Reply:
makes sense. done